### DISCUSSION TITLE
How to insert new fog palette

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#329] [General]
My first impressions are that I'd need to insert a Palette folder in ``Gfx\Sources`` to store the dmp I've created that holds the new palette.

I also suspect I'd need to make some changes to the makefile in that folder to insert the dmp in the GfxInstaller.event and make add a corresponding hook somewhere so that the palette is properly inserted over the original fog palette. 

Though I'd not really sure how to do it, are you able to advise? 😅

**Comment by MokhaLeee** [comment]
The gfx folder is just used for some general image import work. You can still import palettes directly through EA. I took a look at this patch, there are a lot of detailed modifications, so it's not very easy to adapt the program. What kind of skill are you trying to create with it?

**Reply by JesterWizard** [reply]
Magic seal from Kishuna. I've already got the code working, my idea was to have all the tiles within the range of the seal programatically update to be red. I was hoping that would help sidestep the issue of it not playing great with fog.

But failing that, inserting a new fog palette was my backup.
